### ISSUE TITLE
Screen Readers and Other Accessibility in Lessons

### ISSUE THREAD

**Issue by wrightaprilm** [datacarpentry/python-ecology-lesson#367]
Hi folks, 

@Kswhitney raised a question on Twitter about Jupyter Notebooks and screen readers and how well they play together. I did some exploring on my Mac, and have not yet looked at PC yet (my office PC is being replaced, a months-long process for some reason). 

## Here's what I found:

- When used with Safari, Jupyter Notebooks work quite well with the Mac screen reader app. The integration between the Mac screen reader and Safari is clearly a conscious design choice, and there is much finer-grained control of the reader's behavior in Safari.
- When used with Firefox, they do not. [These](https://wiki.mozilla.org/Accessibility/Mobile/ScreenReader) accessibility instructions appear to be out of date for the current FireFox.

## Here is my proposal:

- For now, recommend that Mac users who use a screen reader use Safari as the default web browser for Jupyter. We could add this as a note to the instructor notes. We could also add to the instructor notes instructions on setting Safari as the default browser for Jupyter.

## What needs to be done:

- Test this on PC. I assume FireFox will have similar issues there. 
- Test on Linux.
- Test on Chrome.

## My questions for y'all:

- I feel out of my depth about what assistive technologies are actually being used. If anyone has some readings on what common ones are, I'd be happy to add them to the list o' things to test
-  I feel a little awkward breaking out of the browser agnosticism we've typically maintained. Is this OK? Is it noting the issue and suggesting a specific browser a sufficient solution?

Thanks, y'all, this is important stuff, and I hope we can come to some basic setup that will be workable for folks with this need.

Also tagging in @k8hertweck, since you expressed some interest in this.

**Comment by raynamharris** [comment]
This could be worth posting to Jupyter Discourse or the Jupyter  general mailing list (https://jupyter.org/community) as this is a broadly relevant topic. It might catch the eye of someone with more experience/expertise.

**Comment by wrightaprilm** [comment]
Good idea! Someone tweeted a Discourse thread at me where people were discussing Jupyter accessibility. I'll dig it up and post it there.
